packages/office-ui-fabric-react/src/components/SpinButton/examples/SpinButton.Controlled.Example.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { SpinButton } from 'office-ui-fabric-react/lib/SpinButton';
+
+export interface ISpinButtonControlledExampleState {
+  value: string;
+}
+
+// tslint:disable:jsx-no-lambda
+export class SpinButtonControlledExample extends React.Component<{}, ISpinButtonControlledExampleState> {
+  public constructor(props: {}) {
+    super(props);
+
+    this.state = {
+      value: '37'
+    };
+  }
+
+  public render(): JSX.Element {
+    return (
+      <div style={{ width: '400px' }}>
+        <SpinButton
```

Comment:
Should there be an example where you give a min, max, and step count?

Reply:
That is a good idea.